src/Notifications.jsx

```diff
@@ -95,22 +94,17 @@ export class Notifications extends PureComponent {
     }
 
     componentWillReceiveProps({ isShowing, isVisible, wpcom }) {
-        initAPI(wpcom);
-
-        if (this.props.isShowing && !isShowing) {
-            store.dispatch(actions.ui.closePanel());
-            // unselect the note so keyhandlers don't steal keystrokes
-            reset();
```

Comment:
good call! added back in in e0c7979

I left the `closePanel()` call out of it though